### PR TITLE
Fix runheader pos

### PR DESCRIPTION
**Pull request by MaxNoe**
There was an error in the CORSIKA docs regarding the position of the number of showers.

According to the docs, it was position 94, but it is written to position 93 (start counting at 1).

Heck and Pierog decided to leave as is and update the docs.
These are the changes reflecting that.

Mail from T. Pierog:

> Thanks a lot for this notification. You are completely right. We
will correct this in the next release (not sure whether it is better
to keep it at position 93 as it was before and to keep compatibility
or to really move it to have more space for something which will never
be used probably).

Mail by me:

> I would personally lean towards leaving it at is and just update the documentation of all affected versions (also 7.69, ...).
>
> This would make it easier for everybody reading corsika files (e.g.
simtel array, pycorsikaio, etc.).

Mail by D. Heck

> this I have now performed, i.e. the place 93 of the RUNH block contains the
NSHOW quantity in the program and in the documentation.

### CODE REVIEW

File: corsikaio/subblocks/dtypes.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import numpy as np
+
+
+Field = namedtuple('Field', ['position', 'name', 'shape', 'dtype'])
+
+# set defaults for shape and dtype, this works for the 2 right most attributes
+Field.__new__.__defaults__ = (1, 'f4')
+
+
+def build_dtype(fields, itemsize=4 * 273):
```
Comment (dneise) [inline corsikaio/subblocks/dtypes.py]: Do we really need the developer useing `build_type()` to calculate the itemsize? Can't we deduce it from the fields? What a pity. Looking at the examples of `np.dtype` here: https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.dtype.html

```
Using dictionaries. Two fields named ‘gender’ and ‘age’:

>>> np.dtype({'names':['gender','age'], 'formats':['S1',np.uint8]})
dtype([('gender', '|S1'), ('age', '|u1')])
```

It looks like itemsize is not needed. Without testing any of the following I assume `itemsize` is needed when one wants to use `offsets`. Now looking at this, I believe the way to calculate the offsets will fail as soon as one of the fields is not an `f4` or an `i4` or an `S4` ... So ... hm ... maybe ... this is all fine for here ... just the reusability of this function for other projects is limited...

devs must be a bit careful not to assume too much from this function .. but this is what tests are for .. so fine with me.

Comment (dneise) [reply corsikaio/subblocks/dtypes.py]: Sorry for the noise.

Comment (MaxNoe) [reply corsikaio/subblocks/dtypes.py]: The problem is, when there is unused space at the end of a block. This is the case for run end, event end and event header blocks.

The reading ignores these unused fields, but then the itemsize is not correct. 
As it is known, that each of these corsika blocks is 273 floats, i made it the default.

Comment (MaxNoe) [reply corsikaio/subblocks/dtypes.py]: Let's say we each row is 3 floats, but the last one is unused.
having, only using 
```
>>> dt = np.dtype(dict(names=['x', 'y'], formats=['f4', 'f4']))
>>> dt.itemsize
8
```
This is wrong, as it does not take the third, unused field into account, so we have to explicitly set `itemsize=12`
```

**Pull request merged**